### PR TITLE
Add linters for pixel z / pixel w and for centcomm with two m

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Title

## Why It's Good For The Game

Barely usefull, but centcomm with two m is ugly

## Changelog
:cl:
code: Linters for pixel z / pixel w and for centcomm with two m
/:cl:

### CODE REVIEW

**Comment by SplinterGP** [comment]
wtf is pixel z and w, also pixel_x and pixel_y on maps are fine

**Comment by TiviPlus** [comment]
> wtf is pixel z and w, also pixel_x and pixel_y on maps are fine

read the ref

**Comment by BraveMole** [comment]
> wtf is pixel z and w, also pixel_x and pixel_y on maps are fine

there are zero pixel_x and pixel_y on maps, because you use pixe z and w to do this.

File: _maps/map_files/MarineHQ/MarineHQ.dmm [hunk 1]
```diff
@@ -169,9 +169,7 @@
 /obj/machinery/door/window{
 	dir = 2
 	},
-/obj/machinery/shower{
-	pixel_z = 15
-	},
```
Comment (TiviPlus) [inline _maps/map_files/MarineHQ/MarineHQ.dmm:174]: replace z and w with x and y like they shoud be

File: tools/ci/check_grep.sh [hunk 2]
```diff
@@ -21,6 +21,11 @@ if grep -nP 'step_[xy]' _maps/**/*.dmm;	then
     echo "step_x/step_y variables detected in maps, please remove them."
     st=1
 fi;
+echo "Checking for pixel_[xy]"
+if grep -nP 'pixel_[^xy]' _maps/**/*.dmm;	then
+    echo "pixel_x/pixel_y variables detected in maps, please remove them."
+    st=1
+fi;
```
Comment (TiviPlus) [inline tools/ci/check_grep.sh:28]: x and y?

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: tools/ci/check_grep.sh (hunk 2 above)
Comment (BraveMole) [reply tools/ci/check_grep.sh:28]: ouh, stupid of me

File: tools/ci/check_grep.sh [hunk 3]
```diff
@@ -21,6 +21,11 @@ if grep -nP 'step_[xy]' _maps/**/*.dmm;	then
     echo "step_x/step_y variables detected in maps, please remove them."
     st=1
 fi;
+echo "Checking for pixel_[xy]"
```
Comment (BraveMole) [inline tools/ci/check_grep.sh:24]:
```suggestion
echo "Checking for pixel_[wz]"
```

File: tools/ci/check_grep.sh [hunk 4]
```diff
@@ -21,6 +21,11 @@ if grep -nP 'step_[xy]' _maps/**/*.dmm;	then
...
+if grep -nP 'pixel_[^xy]' _maps/**/*.dmm;	then
+    echo "pixel_x/pixel_y variables detected in maps, please remove them."
```
Comment (BraveMole) [inline tools/ci/check_grep.sh:26]:
```suggestion
    echo "pixel_w/pixel_z variables detected in maps, please remove them."
```

File: tools/ci/check_grep.sh [hunk 5]
```diff
@@ -21,6 +21,11 @@ if grep -nP 'step_[xy]' _maps/**/*.dmm;	then
     echo "step_x/step_y variables detected in maps, please remove them."
     st=1
 fi;
+echo "Checking for pixel_[wz]"
+if grep -nP 'pixel_[^xy]' _maps/**/*.dmm;	then
```
Comment (TiviPlus) [inline tools/ci/check_grep.sh:25]: This still checks for x/y are you sure it works

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**